development/adr/0009-versioning-strategy.md

```diff
@@ -0,0 +1,32 @@
+# **ADR-009: Versioning strategy**
+
+## Status
+
+Proposed
+
+## Context
+
+As a software product, we should establish a sense of versioning.
+With versioning we are able to pin our products as a versioned resource at customer facing deployments
+or for feature planing. This ADR is focused on the Git implementation of our versioning strategy.
+
+## Decision
+
+The base of every versioning strategy is the version number. We are using [Semantic Versioning 2.0.0](https://semver.org/) with version prefix of `v`.
+Our Git strategy is based on the branching and versioning strategy of [Dapr](https://github.com/dapr/dapr).
```

Comment:
Move the dapr line to the bottom somewhere, as a pointer for further reading.
When a new dev reads this, they first check out semver.org and then the dapr repo, without any direction on what to look for